osu.Game.Rulesets.Osu/Skinning/Legacy/LegacyMainCirclePiece.cs

```diff
@@ -35,8 +35,9 @@ public LegacyMainCirclePiece(string priorityLookup = null, bool hasNumber = true
 
         private Drawable hitCircleSprite;
 
-        protected Drawable HitCircleOverlay { get; private set; }
+        protected Container OverlayLayer;
```

Comment:
Is there a reason for removing the `private set` here? Seems to work fine without it.

```suggestion
        protected Drawable HitCircleOverlay { get; private set; }
```

Reply:
none whatsoever other than me not reading the diff before PRing